hiss/team/views.py

```diff
@@ -12,16 +12,22 @@
 from team.forms import CreateTeamForm, JoinTeamForm
 from team.models import Team
 from user.models import User
+from application.models import Application
 
 
-class CreateTeamView(shared_mixins.LoginRequiredAndAppliedMixin, generic.CreateView):
+class CreateTeamView(shared_mixins.UserHasNoTeamMixin, generic.CreateView):
     """
     If the user has applied, creates a Team and adds the User to it.
     """
 
     form_class = CreateTeamForm
     template_name = "team/team_form.html"
 
+    def handle_no_permission(self):
+        if not self.request.user.is_authenticated:
+            return redirect("/accounts/login/?next=/team/")
```

Comment:
Please use `reverse_lazy` here instead of hard-coding the URL.

```suggestion
            return redirect(f"{reverse_lazy('customauth:login')}?next={reverse_lazy('team:create')}")
```